pkg/rancher-desktop/main/tray.ts

```diff
@@ -242,11 +257,18 @@ export class Tray {
    */
   public hide() {
     this.trayMenu.destroy();
-    this.abortController?.abort();
     mainEvents.off('k8s-check-state', this.k8sStateChangedEvent);
     mainEvents.off('settings-update', this.settingsUpdateEvent);
     ipcMainProxy.removeListener('update-network-status', this.updateNetworkStatusEvent);
+    clearInterval(this.fsWatcherInterval);
     clearInterval(this.networkInterval);
+    if (this.runBuildFromConfigTimer) {
+      clearTimeout(this.runBuildFromConfigTimer);
```

Comment:
Do we need to reset it to `null`? Otherwise when we enable it again, we'll never re-setup the timer.